src/PowerShellEditorServices/Services/TextDocument/Handlers/CompletionHandler.cs

```diff
@@ -24,7 +24,8 @@ namespace Microsoft.PowerShell.EditorServices.Handlers
     internal class CompletionHandler : ICompletionHandler, ICompletionResolveHandler
     {
         const int DefaultWaitTimeoutMilliseconds = 5000;
-        private readonly CompletionItem[] s_emptyCompletionResult = Array.Empty<CompletionItem>();
+        private static readonly CompletionItem[] s_emptyCompletionResult = Array.Empty<CompletionItem>();
```

Comment:
It should actually be possible to just use `Array.Empty<T>()` directly (since the purpose of this is to cache internally)

Reply:
fixed